src/test/projects/hello-world-with-test/test/main/java/fr/inira/gforge/spoon/AppTest.java

```diff
@@ -0,0 +1,8 @@
+package fr.inria.gforge.spoon;
+
+public class AppTest {
```

Comment:
the code to be checked should contain a reference to a test dependency such as junit

Reply:
added.